app/src/main/java/com/codingblocks/cbonlineapp/ui/MyCourseCardUi.kt

```diff
@@ -55,48 +58,56 @@ class MyCourseCardUi : AnkoComponent<ViewGroup> {
                 linearLayout {
                     weightSum = 3.0f
                     frameLayout {
-                        linearLayout {
-                            gravity = Gravity.CENTER
-                            orientation = LinearLayout.VERTICAL
-                            elevation = 5f
-                            courselogo = circleImageView {
-                                topPadding = dip(16)
-                                borderColor = Color.parseColor("#ffffff")
-                                borderWidth = dip(2)
-                            }.lparams(width = dip(76), height = dip(76))
-                            courseTitle = textView {
-                                textSize = 20f
-                                typeface = font
-                                textAlignment = View.TEXT_ALIGNMENT_CENTER
-                                textColor = Color.parseColor("#ffffff")
-                            }.lparams(matchParent, wrapContent) {
-                                marginStart = dip(4)
-                                topMargin = dip(8)
-                                marginEnd = dip(4)
-                                bottomMargin = dip(12)
-                            }
-//                            linearLayout {
-//                                courseRatingBar = ratingBar{
-//                                    numStars = 5
-//                                    setIsIndicator(true)
-//                                }.lparams(wrapContent, dip(40))
-//                                courseRatingTv = textView {
-//                                    textSize = 12f
-//                                    typeface = font
-//                                }.lparams(wrapContent, wrapContent) {
-//                                    marginStart = dip(14)
-//                                }
-//                            }.lparams(matchParent, wrapContent) {
-//                                marginStart = dip(8)
-//                                marginEnd = dip(8)
-//
-//                            }
-                        }.lparams(matchParent, matchParent)
-
                         courseCoverImageView = imageView {
                             scaleType = ImageView.ScaleType.CENTER_CROP
                             setImageResource(R.drawable.placeholder_course_cover)
                         }.lparams(width = matchParent, height = matchParent)
+
+                        relativeLayout {
+                            linearLayout {
```

Comment:
@adityastic Please fix this layout ASAP
![Screenshot_20190606-105236](https://user-images.githubusercontent.com/29139786/59008932-70532c80-8849-11e9-8ce8-47e634d9dfed.jpg)

Reply:
@aggarwalpulkit596 I'm not sure what part of it needs to be fixed. Do you want me to revert it to the original state?